docker-compose-services/drupalci-chromedriver/README.md

```diff
@@ -24,8 +24,7 @@ user@demo-web:/var/www/html$ phpunit --verbose -c web/core/phpunit.xml.dist web/
 From @damienmckenna, here's an example run to demonstrate functionality:
 
 ```bash
-$ ddev ssh
-$ phpunit --verbose -c core/phpunit.xml.dist core/modules/system/tests/src/FunctionalJavascript/System/DateFormatTest.php
+$ ddev . phpunit --verbose -c core/phpunit.xml.dist core/modules/system/tests/src/FunctionalJavascript/System/DateFormatTest.php
```

Comment:
I prefer `ddev exec` because although some prefer `ddev .` and it's valid, it's not self-explanatory.

```suggestion
$ ddev exec phpunit --verbose -c core/phpunit.xml.dist core/modules/system/tests/src/FunctionalJavascript/System/DateFormatTest.php
```

I thought you had to edit the phpunit.xml.dist before you could do anything though?

Reply:
Nope, it works out of the box as-is.